jobsdb/jobsdb.go

```diff
@@ -608,17 +616,58 @@ func (jd *HandleT) getDSRangeList(refreshFromDB bool) []dataSetRangeT {
 		//We store ranges EXCEPT for the last element
 		//which is being actively written to.
 		if idx < len(dsList)-1 {
-			jd.assert(minID.Valid && maxID.Valid, fmt.Sprintf("minID.Valid: %v, maxID.Valid: %v. Either of them is false for table: %s", minID.Valid, maxID.Valid, ds.JobTable))
-			jd.assert(idx == 0 || prevMax < minID.Int64, fmt.Sprintf("idx: %d != 0 and prevMax: %d >= minID.Int64: %v", idx, prevMax, minID.Int64))
+			if !minID.Valid || !maxID.Valid {
```

Comment:
Instead of changing these asserts to handle case of migrateTo DS, maybe we can keep migrateTo DS global and disable these checks specifically when ds == migrateTo

Reply:
done